src/js/profile/mobile/widget/Calendar.js

```diff
@@ -0,0 +1,299 @@
+
+				marginRow.style.height = "7px";
+
+				for (var i = 1; i < week + 1; i++) {
+					var row = calendar.insertRow();
+
+					while (day != 0) { // prev disabled days.
+						day = day - 1;
+						leftDays = leftDays - 1;
+						cell = row.insertCell();
+						cell.innerHTML = prevLastDate - day;
+						cell.style.opacity = "10%";
+					}
+					while (leftDays != 0) {
+						if (setDays > prevLastDate) { // next disabled days.
+							cell = row.insertCell();
+							cell.innerHTML = nextMonthDate;
+							cell.style.opacity = "40%";
+							leftDays = leftDays - 1;
+							nextMonthDate = nextMonthDate + 1;
+						} else { // current enabled days.
+							cell = row.insertCell();
+							cell.innerHTML = setDays;
+							cell.style.opacity = "100%";
+							setDays = setDays + 1;
+							leftDays = leftDays - 1;
+						}
+					}
+					leftDays = 7;
+				}
+
+				document.getElementById("yearMonth").innerHTML = fullNameMonth[todayMonth - 1] + " " + todayYear;
+			}
+
+			/**
+			* Moving to another month erases the existing calendar
+			* @method _deleteCalendar
+			* @member ns.widget.mobile.Calendar
+			* @protected
+			*/
+			prototype._deleteCalendar = function () {
+				var calendar = document.getElementById("calendar");
+
+				while (calendar.rows.length > 2) {
+					calendar.deleteRow(2);
+				}
+			}
+
+			/**
+			* Click the arrow on the calendar to move to another month
+			* @method _onClick
+			* @param {Event} event
+			* @member ns.widget.mobile.Calendar
+			* @protected
+			*/
+			prototype._onClick = function (event) {
+				if (event.srcElement.className === "calendarArrowRight") {
+					todayMonth = todayMonth + 1;
+					if (todayMonth === 13) {
+						todayMonth = 1;
+						todayYear = todayYear + 1;
+					}
+					prototype._deleteCalendar();
+					today = new Date(todayYear, todayMonth - 1);
+					prototype._buildCalendar();
+				} else if (event.srcElement.className === "calendarArrowLeft") {
+					todayMonth = todayMonth - 1;
+					if (todayMonth === 0) {
+						todayMonth = 12;
+						todayYear = todayYear - 1;
+					}
+					prototype._deleteCalendar();
+					today = new Date(todayYear, todayMonth - 1);
+					prototype._buildCalendar();
+				} else {
+					event.preventDefault();
+				}
+			};
+
+			prototype._unBindEvents = function (element) {
+				var self = this;
+
+				events.off(element, "vclick", self, false);
+			}
+
+			prototype._bindEvents = function (element) {
+				var self = this;
+
+				events.on(element, "vclick", self, false);
+			};
+
+			prototype.handleEvent = function (event) {
+				var self = this;
+
+				if (event.type === "vclick") {
+					self._onClick(event);
+				} else {
+					event.preventDefault();
+				}
+			};
+
+			prototype._refresh = function () {
+				var self = this;
+
+				self._setValue(self.options.value);
+			}
+
+			prototype._setValue = function (value) {
+				var self = this,
+					options = self.options;
+
+				options.pastSelection = value;
+
+				self._refreshValue();
```

Comment:
Where is the 'prototype._refreshValue()'?

Reply:
not required.
I will remove.

done.